kura/test/org.eclipse.kura.nm.test/src/test/java/org/eclipse/kura/nm/NMDbusConnectorTest.java

```diff
@@ -0,0 +1,897 @@
+        }
+        when(this.mockedNetworkManager.GetAllDevices()).thenReturn(devicePaths);
+    }
+
+    public void givenNetworkConfigMapWith(String key, Object value) {
+        netConfig.put(key, value);
+    }
+
+    public void givenNetworkServiceMockedForUsbInterfaceWithName(String interfaceName) {
+        this.networkService = mock(NetworkService.class);
+
+        UsbNetDevice mockUsbNetDevice = mock(UsbNetDevice.class, RETURNS_DEEP_STUBS);
+
+        when(this.networkService.getUsbNetDevice(interfaceName)).thenReturn(Optional.of(mockUsbNetDevice));
+    }
+
+    public void givenNetworkServiceThatAlwaysReturnsEmpty() {
+        this.networkService = mock(NetworkService.class);
+        when(this.networkService.getUsbNetDevice(any())).thenReturn(Optional.empty());
+    }
+
+    public void whenGetDbusConnectionIsRun() {
+        this.dbusConnectionInternal = this.instanceNMDbusConnector.getDbusConnection();
+    }
+
+    public void whenCheckPermissionsIsRun() {
+        this.instanceNMDbusConnector.checkPermissions();
+    }
+
+    public void whenCheckVersionIsRun() {
+        try {
+            this.instanceNMDbusConnector.checkVersion();
+        } catch (DBusException e) {
+            hasDBusExceptionBeenThrown = true;
+        }
+    }
+
+    public void whenGetInterfacesIsCalled() {
+        try {
+            this.internalStringList = this.instanceNMDbusConnector.getInterfaces();
+        } catch (DBusException e) {
+            hasDBusExceptionBeenThrown = true;
+        }
+    }
+
+    public void whenApplyIsCalled() {
+        try {
+            this.instanceNMDbusConnector.apply();
+        } catch (DBusException e) {
+            hasDBusExceptionBeenThrown = true;
+        }
+    }
+
+    public void whenApplyIsCalledWith(Map<String, Object> networkConfig) {
+        try {
+            this.instanceNMDbusConnector.apply(networkConfig);
+        } catch (DBusException e) {
+            hasDBusExceptionBeenThrown = true;
+        } catch (NoSuchElementException e) {
+            hasNoSuchElementExceptionThrown = true;
+        } catch (NullPointerException e) {
+            hasNullPointerExceptionThrown = true;
+        }
+    }
+
+    public void whenGetInterfaceStatus(String netInterface, NetworkService netService) {
+        try {
+            this.netInterface = this.instanceNMDbusConnector.getInterfaceStatus(netInterface, netService, this.commandExecutorService);
+        } catch (DBusException e) {
+            hasDBusExceptionBeenThrown = true;
+            e.printStackTrace();
+        } catch (NoSuchElementException e) {
+            hasNoSuchElementExceptionThrown = true;
+            e.printStackTrace();
+        } catch (NullPointerException e) {
+            hasNullPointerExceptionThrown = true;
+            e.printStackTrace();
+        } catch (KuraException e) {
+            hasKuraExceptionThrown = true;
+            e.printStackTrace();
```

Comment:
Remove stack trace prints

```suggestion
        } catch (NoSuchElementException e) {
            hasNoSuchElementExceptionThrown = true;
        } catch (NullPointerException e) {
            hasNullPointerExceptionThrown = true;
        } catch (KuraException e) {
            hasKuraExceptionThrown = true;
```

Reply:
Fixed in 1a68e34eb45aa6f7a7fd773e18856c59a29761c7